scalanet/discovery/src/io/iohk/scalanet/discovery/util/package.scala

```diff
@@ -0,0 +1,27 @@
+package io.iohk.scalanet.discovery
+
+import scodec.bits.ByteVector
+import scala.math.Ordering._
+
+
+package object util { 
+
+  private class ByteIterableOrdering extends Ordering[Iterable[Byte]] {
```

Comment:
Can this not be used? 

https://www.scala-lang.org/api/2.13.0/scala/math/Ordering$$Implicits$.html#seqOrdering[CC[X]%3C:scala.collection.Seq[X],T](implicitord:scala.math.Ordering[T]):scala.math.Ordering[CC[T]]

Reply:
Oh I see they renamed the method :-/ 
I wonder if a [trick like this](https://github.com/scala/scala-collection-compat/issues/208#issuecomment-497735669) works in this case as well.

Reply:
Or actually its should work since it's implicit.

Reply:
This works:
```scala
  import Ordering.Implicits._

  implicit val byteOrdering: Ordering[ByteVector] =
    Ordering.by[ByteVector, IndexedSeq[Byte]](_.toIndexedSeq)
```
And the way `ByteVector.toIndexedSeq` works it doesn't create any copies either ;)

Reply:
Yep, that works, I've got confused by renamed method as well :(
But anyway, it works, sorry.